### PR TITLE
Markup update in Appendix section numbering

### PR DESCRIPTION
**Pull request by manuel489**
*As requested in https://github.com/metanorma/bipm-si-brochure/issues/156#issuecomment-933593516*

*Note*: `[%unnumbered]` attributes were applied until sub-sections of level 3.
Level 4 sections were excluded. Let me know if I should apply changes to those as well.

### CODE REVIEW

File: sources/sections-a4-en/03-base-units.adoc [hunk 1]
```diff
@@ -1,20 +1,24 @@
-== Part 3. Historical perspective on the base units(((base unit(s))))
+== Historical perspective on the base units(((base unit(s))))
 
+[%unnumbered]
```
Comment (ronaldtse) [inline sources/sections-a4-en/03-base-units.adoc:3]: All A4 sections are to be numbered, so let's remove `[%unnumbered]`. Only A1 sections (the quoted resolutions/decisions) are to be unnumbered.

File: sources/sections-a4-en/03-base-units.adoc [hunk 2]
```diff
@@ -1,20 +1,24 @@
...
```
Comment (ronaldtse) [inline sources/sections-a4-en/03-base-units.adoc:8]: As above.

File: sources/sections-a4-en/03-base-units.adoc [hunk 3]
```diff
@@ -1,20 +1,24 @@
...
 === Unit of length, metre (((length))) (((metre (stem:["unitsml(m)"]))))
 
 The 1889 definition of the metre, namely, the length of the international prototype of platinum-iridium, was replaced by the 11th CGPM (1960) using a definition based on the wavelength of the radiation corresponding to a particular transition in krypton 86. This change was adopted in order to improve the accuracy with which the definition of the metre could be realized, this being achieved using an interferometer with a travelling microscope to measure the optical path difference as the fringes were counted. In turn, this was replaced in 1983 by the 17th CGPM (Resolution 1, CR, 97, and _Metrologia_, 1984, *20*, 25) with a definition referenced to the distance that light travels in vacuum in a specified interval of time. The original ((international prototype of the metre)), which was sanctioned by the 1st CGPM in 1889 (CR, 34-38), is still kept at the BIPM under conditions specified in 1889. In order to make clear its dependence on the fixed numerical value of the speed of light, stem:[c], the wording of the definition was changed in Resolution 1 of the 26^th^ CGPM (2018).
 
+[%unnumbered]
```
Comment (ronaldtse) [inline sources/sections-a4-en/03-base-units.adoc:13]: As above.

File: sources/sections-a4-en/03-base-units.adoc [hunk 4]
```diff
@@ -1,20 +1,24 @@
...
 (((fundamental constants (of physics))))
 
 By the time of the second verification of national prototypes in 1946 it was found that on average the masses of these prototypes were diverging from that of the international prototype. This was confirmed by the third verification carried out from 1989 to 1991, the median difference being about 25 micrograms for the set of original prototypes sanctioned by the 1st CGPM (1889). In order to assure the long-term stability of the unit of mass, to take full advantage of quantum electrical standards and to be of more utility to modern science, a new definition for the ((kilogram)) based on the value of a fundamental constant, for which purpose the ((Planck constant)) stem:[h] was chosen, was adopted by Resolution 1 of the 26^th^ CGPM (2018).
 
+[%unnumbered]
```
Comment (ronaldtse) [inline sources/sections-a4-en/03-base-units.adoc:21]: As above.

File: sources/sections-a4-en/03-base-units.adoc [hunk 5]
```diff
@@ -25,11 +29,13 @@ By the time of the 8th CGPM (1933) there was a unanimous desire to replace the "
...
 === Unit of thermodynamic temperature, kelvin (((kelvin (stem:["unitsml(K)"])))) (((thermodynamic temperature))) (((triple point of water)))
 
 The definition of the unit of thermodynamic temperature was given by the 10th CGPM (1954, Resolution 3; CR 79) which selected the triple point of water, stem:[ii(T)_("TPW")], as a fundamental fixed point and assigned to it the temperature 273.16 K, thereby defining the kelvin. The 13th CGPM (1967-1968, Resolution 3; CR, 104 and _Metrologia_, 1968, *4*, 43) adopted the name kelvin, symbol stem:["unitsml(K)"], instead of "degree kelvin", symbol stem:["unitsml(degK)"], for the unit defined in this way.((("water, isotopic composition")))
 However, the practical difficulties in realizing this definition, requiring a sample of pure water of well-defined isotopic composition and the development of new primary methods of thermometry, led to the adoption of a new definition of the kelvin based on a fixed numerical value of the ((Boltzmann constant)) stem:[k]. The present definition, which removed both of these constraints, was adopted in Resolution 1 of the 26^th^ CGPM (2018).
 
+[%unnumbered]
```
Comment (ronaldtse) [inline sources/sections-a4-en/03-base-units.adoc:38]: As above.

File: sources/sections-a4-en/03-base-units.adoc [hunk 6]
```diff
@@ -25,11 +29,13 @@ By the time of the 8th CGPM (1933) there was a unanimous desire to replace the "
 
 However the 1948 definition of the ampere(((ampere (stem:["unitsml(A)"])))) proved difficult to realize and practical quantum standards (based on Josephson and quantum-Hall effects), which link both the volt and the ohm to particular combinations of the ((Planck constant)) stem:[h] and ((elementary charge)) stem:[e], became almost universally used as a practical realization of the ampere(((ampere (stem:["unitsml(A)"])))) through Ohm's law (18th CGPM (1987), Resolution 6, CR 100). As a consequence, it became natural not only to fix the numerical value of stem:[h] to redefine the ((kilogram)), but also to fix the numerical value of stem:[e] to redefine the ampere(((ampere (stem:["unitsml(A)"])))) in order to bring the practical quantum electrical standards into exact agreement with the SI. The present definition based on a fixed numerical value for the ((elementary charge)), stem:[e], was adopted in Resolution 1 of the 26^th^ CGPM (2018).
 
+[%unnumbered]
```
Comment (ronaldtse) [inline sources/sections-a4-en/03-base-units.adoc:32]: As above.

File: sources/sections-a4-en/03-base-units.adoc [hunk 7]
```diff
@@ -46,6 +52,7 @@ Thus, this definition of the mole(((mole (stem:["unitsml(mol)"])))) was dependen
 
 The numerical value of the ((Avogadro constant)) defined in this way was equal to the number of atoms in 12 grams of ((carbon 12)). However, because of recent technological advances, this number is now known with such precision that a simpler and more universal definition of the mole has become possible, namely, by specifying exactly the number of entities in one mole of any substance, thus fixing the numerical value of the ((Avogadro constant)). This has the effect that the new definition of the mole and the value of the ((Avogadro constant)) are no longer dependent on the definition of the ((kilogram)). The distinction between the fundamentally different quantities '((amount of substance))' and 'mass' is thereby emphasized. The present definition of the mole based on a fixed numerical value for the ((Avogadro constant)), stem:[ii(N)_"A"], was adopted in Resolution 1 of the 26^th^ CGPM (2018).
 
+[%unnumbered]
```
Comment (ronaldtse) [inline sources/sections-a4-en/03-base-units.adoc:55]: As above.

File: sources/sections-a4-en/03-base-units.adoc (hunk 1 above)
Comment (manuel489) [reply sources/sections-a4-en/03-base-units.adoc]: Done.

File: sources/sections-a4-en/03-base-units.adoc (hunk 7 above)
Comment (manuel489) [reply sources/sections-a4-en/03-base-units.adoc]: Done.

File: sources/sections-a4-en/03-base-units.adoc (hunk 6 above)
Comment (manuel489) [reply sources/sections-a4-en/03-base-units.adoc]: Done.

File: sources/sections-a4-en/03-base-units.adoc (hunk 5 above)
Comment (manuel489) [reply sources/sections-a4-en/03-base-units.adoc]: Done.

File: sources/sections-a4-en/03-base-units.adoc (hunk 4 above)
Comment (manuel489) [reply sources/sections-a4-en/03-base-units.adoc]: Done.

File: sources/sections-a4-en/03-base-units.adoc (hunk 3 above)
Comment (manuel489) [reply sources/sections-a4-en/03-base-units.adoc]: Done.

File: sources/sections-a4-en/03-base-units.adoc (hunk 2 above)
Comment (manuel489) [reply sources/sections-a4-en/03-base-units.adoc]: Done.

**Review by ronaldtse — APPROVED** [review]

**Comment by ronaldtse** [comment]
Thanks @manuel489 !

**Pull request merged**